### DISCUSSION TITLE
[Feature Request] save/print/export -contrast-stretch values for reuse on other images

### DISCUSSION THREAD

**Discussion by memphis-10** [ImageMagick/ImageMagick#4028] [Development]
### Edited to reflect request

Hoping to get implementation that will allow users to save -contrast-stretch values from one image to reuse on other images.

There is a discussion in Digital Image Processing with workarounds here:
https://github.com/ImageMagick/ImageMagick/discussions/4025

**Comment by dlemstra** [comment]
I changed your issue into a discussion @memphis-10.

**Reply by memphis-10** [reply]
Thank you.  I will edit the original post and move it to development.  I already have a discussion with workarounds suggested in Digital Image Processing
